Gems/AWSClientAuth/Code/Source/AWSClientAuthSystemComponent.h

```diff
@@ -17,15 +17,14 @@
 
 namespace AWSClientAuth
 {
-    //! Gem System Component. Responsible for instantiating and managing Authentication and Authroziation Controller
+    //! Gem System Component. Responsible for instantiating and managing Authentication and Authorization Controller
     class AWSClientAuthSystemComponent
         : public AZ::Component
         , public AWSCore::AWSCoreNotificationsBus::Handler
         , public AWSClientAuthRequestBus::Handler
     {
     public:
-
-        virtual ~AWSClientAuthSystemComponent() = default;
+        ~AWSClientAuthSystemComponent() override = default;
```

Comment:
what's the impact of no longer making this virtual?

Reply:
Theres no impact, destructor is still virtual, its declared virtual in base classes but now we correctly mark it as overriding the base class destructor.